scripts/setup.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+pip install -r requirements.txt
+
+git clone git@github.com:NVIDIA/apex.git
+cd apex
+git checkout 1b903852aecd388e10f03e470fcb1993f1c871dd
+python setup.py install
```

Comment:
What do you think of changing this line to what is mentioned in the apex readme:
"pip install -v --no-cache-dir --global-option="--cpp_ext" --global-option="--cuda_ext" ./"

Reply:
I think the command (suggested) is taken from the latest version of apex code. I used the command (committed)  from an [older version of apex](https://github.com/NVIDIA/apex/tree/1b903852aecd388e10f03e470fcb1993f1c871dd). The older version is the version referenced in the README of this repo. Since we are using the older version, maybe its best to stick to the install instructions corresponding to that version.